src/index.js

```diff
@@ -1,9 +1,11 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import Home from './pages/Home';
-import './index.css';
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+import Home from './pages/Home'
+
+import './index.scss'
 
 ReactDOM.render(
   <Home />,
   document.getElementById('root')
```

Comment:
I noticed this just now, but we should have another component (`App`, `MainRouter`, `AppRouter`) that has [routing logic](https://reactrouter.com/web/example/basic). So that it renders a `Home` only if the path is `/`. This also allows more pages to be added in the future!